<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoProgramUIClass</class>
 <widget class="QDialog" name="AutoProgramUIClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iAssist - Program iButtons</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStatus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Button Nr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtButtonNr">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Button ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtButtonID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecFootprint">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>150</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIncFootprint">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblFootprint">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIncFootprintMore">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecFootprintMore">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>40</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnStart</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>btnDecFootprintMore</tabstop>
  <tabstop>btnDecFootprint</tabstop>
  <tabstop>btnIncFootprint</tabstop>
  <tabstop>btnIncFootprintMore</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
